sw/device/tests/BUILD

```diff
@@ -4615,6 +4615,7 @@ test_suite(
             otp = test_cfg["otp"],
             tags = [
                 "lc_{}".format(test_cfg["lc_state"]),
+                "broken",
```

Comment:
I think only the `delayed_enabled`, the other variants of this test is working fine.

Reply:
fixed